test/unit/gen/certificaterequest.go

```diff
@@ -107,6 +107,12 @@ func SetCertificateRequestName(name string) CertificateRequestModifier {
 	}
 }
 
+func SetCertificateRequestUsages(usages ...v1alpha1.KeyUsage) CertificateRequestModifier {
```

Comment:
Used?

Reply:
🤷‍♂ I think it's useful to have even if it isn't used (and also: probably should be used in a test somewhere 😅)